code.sh

```diff
@@ -14,6 +15,10 @@ if [ ! -f ${FIRST_RUN} ]; then
   touch ${FIRST_RUN}
 fi
 
+if [ "$XDG_SESSION_TYPE" = "wayland" ]; then
+  WAYLAND_OPTS="--ozone-platform-hint=auto"
+fi
```

Comment:
The outer check isn't necessary -- --ozone-platform-hint=auto means "Check if XDG_SESSION_TYPE=wayland, if so, use wayland, otherwise use x11".